### PR TITLE
Feature/main architecture

### PR DESCRIPTION
**Pull request by JaimeMathias**
Implemented a Entity for the Environment Variables, made a Service to get them from Environment, initialized and logged the variables on console at Main(). Also done a unit tests that verifies if the variable from Environment is coming correctly and another to test when that variable is null.

Unit testing results:
+--------+--------+--------+--------+
| Module | Line   | Branch | Method |
+--------+--------+--------+--------+
| App    | 56,55% | 70,45% | 90%    |
+--------+--------+--------+--------+

### CODE REVIEW

**Comment by silva-willian** [comment]
Colocar na restrição do git as pastas .vscode/,  **/bin e **/obj e o arquivo .env

File: App/Program.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            return 0;
+        }
+
+        public static void PrintData(StructuralData data) 
+        {
+            System.Console.WriteLine("Language: " + data.Language);
+            System.Console.WriteLine("RoutePattern: " + data.RoutePattern);
+            System.Console.WriteLine("Versioned: " + data.Versioned);
+            System.Console.Write("HttpVerbs: ");
+            foreach (var value in data.HttpVerbs) {
```
Comment (silva-willian) [inline App/Program.cs:46]: Não usar Console.WriteLine, o .net core tem uma biblioteca especifica para log, utilizar ela

**Comment by silva-willian** [comment]
Coloquem na restrição do git o arquivo de coverage dos testes

**Comment by silva-willian** [comment]
Cade o readme do projeto ?
